src/oryx/install.sh

```diff
@@ -236,16 +234,19 @@ fi
 if [[ "${PINNED_SDK_VERSION}" != "" ]]; then
     rm -f ${GIT_ORYX}/global.json
     rm -rf /usr/share/dotnet/sdk/$PINNED_SDK_VERSION
-
-    # Extract the major, minor version and the first digit of the patch version
-    MAJOR_MINOR_PATCH1_VERSION=${PINNED_SDK_VERSION%??}
-    rm -rf /usr/share/dotnet/shared/Microsoft.NETCore.App/$MAJOR_MINOR_PATCH1_VERSION
-    rm -rf /usr/share/dotnet/shared/Microsoft.AspNetCore.App/$MAJOR_MINOR_PATCH1_VERSION
-    rm -rf /usr/share/dotnet/templates/$MAJOR_MINOR_PATCH1_VERSION
+    NEW_RUNTIME_VERSIONS=$(dotnet --list-runtimes | awk '{print $2}' | sort | uniq)
+    if [ -n "${RUNTIME_VERSIONS:-}" ]; then
```

Comment:
This RUNTIME_VERSIONS variable will be set only for images with dotnet even before oryx feature installation
So for images without DOTNET this variable will not be set .
This throws issue stating unbound variable.
"${RUNTIME_VERSIONS:-} This syntax will check RUNTIME_VERSIONS is unset or null if unset or null it will use an empty string as the value.